test/compile/scale.test.ts

```diff
@@ -331,7 +331,7 @@ describe('Scale', function() {
 
     describe('for ordinal', function() {
       it('should return correct domain with the provided sort property', function() {
-        const sortDef = {op: 'min', field:'Acceleration'};
+        const sortDef = {op: 'min' as 'min', field:'Acceleration'};
```

Comment:
Well. Yes and no. Either the `const sortDef` needs a type annotation of the correct type (which is.... `FieldDef`? Maybe?) to cause `'min'` to be contextually typed as a string literal... or you can just cast it directly to the appropriate string literal type.